CIP-0013/CIP-0013.md

```diff
@@ -1,27 +1,37 @@
 ---
 CIP: 13
 Title: Cardano URI Scheme
-Authors: Sebastien Guillemot <sebastien@emurgo.io>, Vicente Almonacid <vicente@emurgo.io>
+Authors: Sebastien Guillemot <sebastien@emurgo.io>, Vicente Almonacid <vicente@emurgo.io>, Robert Phair <rphair@cosd.com>
 Comments-URI:
 - https://github.com/Emurgo/EmIPs/pull/2
 - https://forum.cardano.org/t/cip-cardano-payment-uri-scheme/41457
+- https://github.com/cardano-foundation/CIPs/pull/25
+- https://github.com/cardano-foundation/CIPs/pull/61
+- https://forum.cardano.org/t/cip-stake-uri-scheme-for-pools-delegation-portfolios/40594
 Status: Draft
 Type: Informational
-Created: 2020-10-20
+Created: 2020-09-22
 License: CC-BY-4.0
 ---
 
 # Abstract
 
-This proposal describes a basic URI scheme to handle ADA transfers, as well
-as possible approaches for a multiplatform implementation.
+This proposal describes a basic URI scheme to handle ADA transfers and links to stake pools, as well as possible approaches for a multiplatform implementation.
```

Comment:
> as well as possible approaches for a multiplatform implementation.

sorry, that can be deleted. This is a copy+paste mistake I made in the original CIP

Reply:
I've deleted that phrase.  Also the correct spelling is "Ada" in this context: not the capitalised trading symbol but rather the proper name of the currency itself, also opposed to "ada" which is the common name of the currency units... I should have noticed that before.